tests/integration/test_bayesian_optimization.py

```diff
@@ -50,6 +50,8 @@
 from trieste.space import Box, SearchSpace
 from trieste.types import State, TensorType
 
+tf.keras.backend.set_floatx("float64")
```

Comment:
Changes in this file are likely stale and shouldn't be necessary

Reply:
what do you mean?

Reply:
Removed